test-lib.sh

```diff
@@ -538,8 +578,14 @@ EOF
     else
         echo "CMD /usr/libexec/s2i/run" >>"$df_name"
     fi
+    if $ca_file; then
+        # Check if -v parameter is present in s2i_args and add it into docker build command
+        mount_ca_option=$(echo "$s2i_args" | grep -o -e '\(-v\)[[:space:]]\.*\S*')
+    else
+        mount_ca_option=""
+    fi
```

Comment:
If `$ca_file` does not exist a volume will not be mounted into the build containers. Why should the volume argument be limited in such a way?

Reply:
Well, because it is relevant only for mounting CA file. But I can rename it to `mount_option` and we can use it always.

Reply:
Yeah I think that would be better. The volume parameter for ca_file mount is controlled by `ct_build_s2i_npm_variables` so it would not make much sense to have it in two places anyway.